### PR TITLE
add NEXT_MAJOR to PHPDoc comment

### PR DESCRIPTION
**Pull request by OskarStark**
otherwise StyleCi does not pass (https://github.com/sonata-project/SonataAdminBundle/pull/4109)

https://styleci.io/analyses/8LmYmw

### CODE REVIEW

File: project/CONTRIBUTING.md [hunk 1]
```diff
@@ -180,9 +180,9 @@ namespace Foo;
 
 interface BarInterface
 {
-
-  // NEXT_MAJOR: Uncomment this method
   /**
+   * NEXT_MAJOR: Uncomment this method
```
Comment (Soullivaneuh) [inline project/CONTRIBUTING.md]: > and remove this line.

?

**Comment by greg0ire** [comment]
LGTM

File: project/CONTRIBUTING.md (hunk 1 above)
Comment (greg0ire) [inline project/CONTRIBUTING.md]: This is a bit overkill IMO.

Comment (OskarStark) [inline project/CONTRIBUTING.md]: > This is a bit overkill IMO.

yes, this is too, much, the guy who will release the next major knows how to handle the `NEXT_MAJOR` tags

Comment (Soullivaneuh) [inline project/CONTRIBUTING.md]: > the guy who will release the next major knows how to handle the NEXT_MAJOR tags

Or not. This is not the role of the releases guy to remove the NEXT_MAJOR comments.

Comment (OskarStark) [inline project/CONTRIBUTING.md]: Ok but this will be done as a PR against master/major branch and in this case we will review this

**Pull request merged**